src/commands/part.cpp

```diff
@@ -23,42 +20,47 @@
 #include "../../includes/numericReplies.hpp"
 #include "../../includes/utils.hpp"
 
+void Server::quitChannel(int fd, Channel *channel, Client *client) {
+  sendPartMessageToClient(fd, client->getNickname(), channel->getName());
+  // broadcastPartMessage(fd, client->getNickname(), channel->getName());
+  client->decrementChannelsCount();
+  channel->removeClientFromChannelMap(client);
+  if (channel->getChannelOperators().find(fd) !=
+      channel->getChannelOperators().end()) {
+    channel->removeOperator(client);
+  }
+  broadcastInChannel(*client, *channel, "PART", " say goodbye!");
+}
```

Comment:
Remove a space before `say goodbye!`

Hexchat currently prints 
`lala (lala) has left ( say goodbye!)`
There is an extra space after `(`.